roles/setup_gitea/templates/namespace.j2

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "{{ sg_namespace }}"
```

Comment:
Missed line break at the end of the file

Reply:
Done